core/templates/pages/exploration-player-page/services/prediction-algorithm-registry.service.ts

```diff
@@ -57,7 +59,7 @@ export class PredictionAlgorithmRegistryService {
   testOnlySetPredictionService(
       algorithmId: string, dataSchemaVersion: number,
       service: PredictionService): void {
-    if (!this.algorithmIdPredictionServiceMapping.has(algorithmId)) {
+    if (!this.algorithmIdPredictionServiceMapping.get(algorithmId)) {
```

Comment:
same as above

Reply:
done

Reply:
Can't separate out here, logic will behave differently as we're using `.set`